src/plopp/graphics/fig1d.py

```diff
@@ -48,6 +48,10 @@ class Figure1d(BaseFig):
     ax:
         If supplied, use these axes to create the figure. If none are supplied, the
         canvas will create its own axes.
+    form:
```

Comment:
```suggestion
    format:
```
would be much easier to remember, as it matches the `savefig` param name

Reply:
I avoided format because it is a special name.
![Screenshot at 2022-11-08 12-21-59](https://user-images.githubusercontent.com/39047984/200551584-1ad95ede-114b-4450-96cc-6996ab23ed71.png)

So doing `self._format = format` is not the best.
Thoughts?

Reply:
Personally I think it is fine, @jl-wynen?